packages/fullstack/src/hooks/server_future.rs

```diff
@@ -60,7 +58,7 @@ where
         }
     }
 
-    if dependencies.clone().apply(&mut state.dependencies) || state.needs_regen.get() {
+    if dependencies.clone().apply(&mut Vec::new()) || state.needs_regen.get() {
```

Comment:
This will break memorization because the vec will always be empty which will cause apply to return true and always rerun. You could pull it out into a hook like #1666

Reply:
trying to factor it out but didn't remember exactly what the steps were, thank you, will fix now